src/main/java/org/mockito/exceptions/base/MockitoInitializationException.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2007 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+
+package org.mockito.exceptions.base;
+
+public class MockitoInitializationException extends RuntimeException {
```

Comment:
I suggest `MockitoMockMakerException` and to move it in the same mockmaker package since the exception is not that public and is specialized for constructors. Thought ?